app/controllers/EditionsController.scala

```diff
@@ -24,6 +24,7 @@ import util.{SearchResponseUtil, UserUtil}
 import scala.jdk.CollectionConverters._
 import scala.concurrent.ExecutionContext
 import scala.util.Try
+import model.editions.client.EditionsFrontendCollectionWrapper
```

Comment:
Necessary as code in `editions/client` is now in the `editions.client` package, on the premise that the package structure should reflect the file structure.